receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/metadata.yaml

```diff
@@ -37,3 +37,11 @@ metrics:
       aggregation: cumulative
       monotonic: false
     attributes: [device, mode, mountpoint, type, state]
+
+  system.filesystem.utilization:
+    enabled: false
+    description: Percentage of filesystem bytes used.
```

Comment:
According to 
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming 

> utilization - an instrument that measures the fraction of usage out of its limit should be called entity.utilization. For example, system.memory.utilization for the fraction of memory in use. Utilization values are in the range [0, 1].

So this metrics hast to be fraction of 1 and cannot be percentage unless you want to propose a change to the spec

Reply:
Good point, it is actually 0 -> 1 float value. I will update description 👍

Reply:
Changed to `Fraction of filesystem bytes used.`